### DISCUSSION TITLE
After installing HaC Mini, Windows Driver and Support Assistant detects my NUC as "Intel® Client Systems Macmini8,1"

### DISCUSSION THREAD

**Discussion by Zhonecage** [osy86/HaC-Mini#612] [Q&A]
I installed HaC mini on my NUC8i7HNK and dual booted with a second SSD for Windows. Everything works fine (except Secure Boot, but that's another story) but there is one cosmetic issue: Intel's Windows Driver and Support Assistant website detects my NUC as an "Intel® Client Systems Macmini8,1" while on Windows 10. How is that possible and how do I change it back to being detected as a NUC8i7HNK system?

**Comment by osy** [comment]
It’s because you booted from opencore. If you want easy dual booting that’s how it is. If you know how, you can boot directly without opencore and it won’t show that.
